packages/polaris-viz/src/components/StackedAreaChart/utilities/yAxisMinMax.ts

```diff
@@ -21,5 +22,12 @@ export function yAxisMinMax(stackedValues: StackedSeries[]) {
     ),
   );
 
+  if (minY === Infinity || maxY === -Infinity) {
```

Comment:
@envex is this the handling we need here now that we don't default to Infinity internally?